xcape/core.py

```diff
@@ -129,40 +129,78 @@ def calc_cape(p, t, td, ps, ts, tds, **kwargs):
     zMUlev : array-like
         height of MUlev (m)
      """
-    
+    if len(args)>1:
+        print('too many optional arguments')
+        
+    elif kwargs['vertical_lev']!=True:
+        print('please select type of vertical grid, pressure level, or model level')
+        
```

Comment:
if not defined, the user has to define it.

Reply:
Raise error, don't print.

I would recommend a more explicit check and error message

```python
allowed_vertical_levs = ['sigma', 'pressure']
if kwargs['vertical_lev'] not in allowed_vertical_levs:
    raise ValueError(f"`vertrical_lev` must be one of: {allowed_vertical_levs}")